docs/src/main/sphinx/connector/base-jdbc.rst

```diff
@@ -0,0 +1,194 @@
+    - Supports case insensitive database and collection names
+    - False
+  * - ``case-insensitive-name-matching.cache-ttl``
+    -
+    - 1 minute
+  * - ``jdbc-types-mapped-to-varchar``
+    - Allow forced mapping of comma separated lists of data types to convert to
+      unbounded ``VARCHAR``
+    -
+  * - ``metadata.cache-ttl``
+    -  Duration for which meta data, including table and column statistics, is
+       cached
+    - 0 - disabled caching
+  * - ``metadata.cache-missing``
+    - Cache the fact that meta data, including table and column statistics, is
+      not available
+    - False
+
+.. _base-jdbc-performance:
+
+Performance
+-----------
+
+The JDBC connector base includes a number of features to improve connector
+performance. These features need to be extended in the connector to be available
+as user facing features.
+
+.. _base-jdbc-table-statistics:
+
+Table statistics
+^^^^^^^^^^^^^^^^
+
+Connectors supports :doc:`table and column statistics </optimizer/statistics>`.
+They are used for :doc:`cost based optimizations
+</optimizer/cost-based-optimizations>` to improve query processing performance
+based on the actual data in the data source.
+
+Collection of statistics varies for each data source.
+
+Table and column statistics are enabled by default. You can disable it for a
+catalog by setting ``statistics.enabled`` to ``false``. The ``metadata``
+properties from the preceding section further control the statistics caching.
+
+The statistics can be viewed in Trino using :doc:`/sql/show-stats`.
+
+.. _base-jdbc-join-pushdown:
+
+Join pushdown
+^^^^^^^^^^^^^
+
+Join pushdown allows the connector delegate the table join operation to the
+underlying data source. This can result in performance gains and allows Trino to
+perform the remaining query processing on a smaller amount of data.
+
+The specifics for the supported pushdown of table joins varies for each data
+source and therefore for each connector.
+
+.. _base-jdbc-topn-pushdown:
+
+TopN pushdown
+^^^^^^^^^^^^^
+
+TopN pushdown, also called limit pushdown, refers to the common SQL clauses for
+``SELECT`` statements ``TOP N`` or ``LIMIT N``. These equivalent clauses reduce
+the number of rows returned by a query to ``N`` rows. A pushdown of this clause
+to the underlying data source can improve the performance of the query and
+significantly reduce the amount of data transferred from the data source to
+Trino.
+
+Implementation and support is connector specific since different data sources
+support different SQL syntax.
+
+.. _base-jdbc-aggregation-pushdown:
+
+Aggregation pushdown
+^^^^^^^^^^^^^^^^^^^^
+
+Some :doc:`aggregation functions can be pushed down </optimizer/pushdown>` to
+the data source for processing. This can result in significant performance gain.
+Each data source supports different aggregation functions, and pushdown is
+therefore different for each connector.
+
+.. _base-jdbc-connection-pooling:
+
+JDBC connection pooling
```

Comment:
I don't think there is a generic feature like this.

AFAIK we support pooling in Oracle connector only.

Reply:
It might be a starburst only feature.. then I will take this out here. Can you confirm?

Reply:
I see it for [Greenplum](https://docs.starburst.io/latest/connector/starburst-base-jdbc.html#jdbc-connection-pooling). Still might be Starburst only.

Reply:
Confirmed in Trino it is limited to Oracle only.